databrowser/src/pages/datasets/DetailViewPage.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <AppLayout>
+    <ContentArea>
+      <TabGroup>
+        <TabList
+          class="flex relative py-7 space-x-4 border-t border-b border-gray-300"
+        >
+          <TabItem>Table view</TabItem>
+          <TabItem>Detail view</TabItem>
+          <TabItem>Raw Data</TabItem>
+          <div class="absolute right-0">
+            <LanguagePicker />
+          </div>
+        </TabList>
+        <TabPanels>
+          <TabPanel>
+            <div>
+              Dataset Detail Page (type = {{ $route.params.datasetType }}, ID =
+              {{ $route.params.datasetId }})
+            </div>
+          </TabPanel>
+          <TabPanel>Details</TabPanel>
+          <TabPanel>Raw</TabPanel>
+        </TabPanels>
+      </TabGroup>
+    </ContentArea>
+  </AppLayout>
+</template>
+
+<script class="ts">
```

Comment:
If think there is a typo, instead of `<script class="ts">` it should be `<script lang="ts">`